docs/k8s_example.yml

```diff
@@ -85,6 +85,19 @@ spec:
     role: leader
     tier: backend
 
+---
+apiVersion: "v1"
+kind: "ConfigMap"
+metadata:
+  name: "custom"
```

Comment:
Hmm we maybe should name this `customization`, `ots-customize` or something… That way looking at the namespace wouldn't yield that "The heck is that?!?" feeling… :D

Reply:
Good point. Fixed in e8c786a520019a2f7ac6f09b33ba964d2991a4e7.

Reply:
Yipp I do too (`{{ .Release.Namespace }}`) but still seeing a `custom` CM in a namespace would make me think "who created a random CM and why?!?" before noticing that's intended config… :D